src/app.jsx

```diff
@@ -210,6 +214,7 @@ export const Application = () => {
                     { title: _("Edit properties"), onClick: _editProperties },
                     { title: cockpit.format(_("Rename $0"), selectedContext?.type), onClick: _renameItem },
                     { type: "divider" },
+                    { title: cockpit.format(_("Duplicate $0"), selectedContext?.type), onClick: _duplicateItem },
```

Comment:
I would use the 'Copy to...' label that the nautilus is using and allow the user to select the target directory.

Reply:
So this should use the `FileAutoComplete` component (like in creating link)? Because it shows all files, while we want directories only

Reply:
True, I think the FileAutocomplete component would need extension to support showing only specific type (file vs directory)

Reply:
but for now it should be fine to introduce this with a plain text input